penpost/penpost-web
# Gaj/testing/unstubbed components

#### Purpose:
 > Check all that apply.

### What does this PR do?
 > Clearly and concisely describe changes made. 

Adds tests for userlanding and for create postcard.

Issues closed #40 #41 

### Problems Encountered & Resulting Solutions  
> State any problems encountered 

Serious problems were met when trying to stub graphQL and Apollo. 

> Describe your code changes in detail for reviewers.

Due to the hours spent trying to understand stubbing, we decided to test for base functionality instead. 

### Where should the reviewer start?
> What file(s) and line(s) are where changes are made?

Please see Cypress files. 

### How should this be tested?
> What steps need to be taken to confirm functionality? 

please run Cypress locally with  the localhost running in the background. 

### Steps To Take
> Any questions and possible steps moving forward.

Finish postcard preview & submission testing and look into stubbing further.
